open-company/open-company-web
# Mobile edit restyle

Card: https://trello.com/c/2UbXQncp

From Paper doc: 


> Update the Cmail UI
>    Change header to include pencil icon + new post text
>    Post title only appears in top bar when minimized to bottom of browser
>    Move “record video” button and “must see” to bottom bar
>    Clicking “must see” icon changes it to blue, as well as the top bar
> New mobile Web video flow
> Update mobile post creation

Design:
desktop changes:
https://app.goabstract.com/share/27655f84-0762-43f8-9220-d247cc09d036
https://www.dropbox.com/s/a9y9l5ljoc4frks/Screenshot%202018-09-19%2015.31.13.png?dl=0
https://www.dropbox.com/s/2ju540a3zzbep9o/Screenshot%202018-09-19%2014.17.33.png?dl=0

mobile changes:
https://projects.invisionapp.com/share/5QO56G074Y2
https://projects.invisionapp.com/share/AYO56CL2XPT


NB: mobile video is on hold for now due to some issues on iOS and other devices.

To test:
- add a new draft from desktop
- publish add a new post from desktop
- edit a draft on desktop
- publish edit a draft on desktop
- add a video to a post on desktop
- remove video from a post on desktop
- add a new draft from mobile
- publish add a new post from mobile
- edit a draft on mobile
- publish edit a draft on mobile
- now test the sections picker on mobile and make sure the post is added/saved to the section selected
